src/Components/Server/src/ComponentsHub.cs

```diff
@@ -92,7 +94,7 @@ private async void CircuitHost_UnhandledException(object sender, UnhandledExcept
             try
             {
                 _logger.LogWarning((Exception)e.ExceptionObject, "Unhandled Server-Side exception");
-                await circuitHost.Client.SendAsync("JS.Error", e.ExceptionObject);
+                await circuitHost.Client?.SendAsync("JS.Error", e.ExceptionObject);
```

Comment:
When will this ever be null for the hub/circuit case?

Reply:
Removed